packages/mastering/src/mastering/index.ts

```diff
@@ -139,6 +139,19 @@ function assertExamIsValid(doc: Document): Document {
     }
   }
 
+  const examCode = doc.find<Element>('string(/e:exam/@exam-code)', ns).toString()
```

Comment:
Turha käyttää kallista XPathia tähän.

```suggestion
  const examCode = doc.root()?.attr('exam-code')?.value()
```